mqml/parameter/conductresist.py

```diff
@@ -67,23 +67,20 @@ def desoverh_fpm(self, lockin_param1: Parameter, lockin_param2: Parameter) -> fl
         except ZeroDivisionError:
             warnings.warn('the denominator iz zero, returning NaN')
             return np.nan
-        except TypeError as typerror:
-            warnings.warn('Amplification and/or voltaga divisions are not set. Set them and try again.')
-            raise typerror
+        except:
```

Comment:
I think its better to do `except TypeError:` ...

That way you ensure that you only catch that one error type and not something unrelated